src/azure-cli/azure/cli/command_modules/advisor/custom.py

```diff
@@ -17,37 +17,50 @@ def list_recommendations(client, ids=None, resource_group_name=None,
 
 def disable_recommendations(client, ids=None, recommendation_name=None,
                             resource_group_name=None, days=None):
+    if recommendation_name is None and ids is None:
+        from knack.prompting import prompt_y_n
+
+        if not prompt_y_n("\nAre you sure you want to disabled all recommendation?"):
```

Comment:
I think the text should be "Are you sure you want to disable all recommendations?"